packages/inventory-compliance/src/Utilities/hooks/TableTools/useTableTools.js

```diff
@@ -0,0 +1,70 @@
+import { useEffect } from 'react';
+import useFilterConfig from './useFilterConfig';
+import useTableSort from './useTableSort';
+import usePaginate from './usePaginate';
+import useRowsBuilder from './useRowsBuilder';
+import useBulkSelect from './useBulkSelect';
+
+export const useIdentify = (items, options = {}) => {
```

Comment:
I wonder if this would make sense to be implemented as/with and/or even expose it to have other hooks could rely on that to manage items.